source/_includes/puppet-collections/_rsk_import_key.md

```diff
@@ -0,0 +1,6 @@
+    gpg: requesting key 4BD6EC30 from hkp server keys.gnupg.net
```

Comment:
If we're updating these files, can we update them to the new key? see https://github.com/puppetlabs/puppet-docs/pull/686 for those changes